media-video/ffmpeg/ffmpeg-9999.ebuild

```diff
@@ -220,6 +220,10 @@ RDEPEND="
 	mmal? ( media-libs/raspberrypi-userland )
 	modplug? ( >=media-libs/libmodplug-0.8.8.4-r1[${MULTILIB_USEDEP}] )
 	openal? ( >=media-libs/openal-1.15.1[${MULTILIB_USEDEP}] )
+	opencl? (
+	>=app-eselect/eselect-opencl-1.1.0-r4
+		media-libs/opencl[${MULTILIB_USEDEP}]
```

Comment:
do you mean virtual/opencl ?
might require some more restriction: last time I tried this didn't work with e.g. nvidia opencl implementation

indentation is off here btw

Reply:
Ugh, I stared at that so long and didn't notice I was using media-libs when I definitely meant virtual. 🤦‍♂️ Thanks!
According to https://trac.ffmpeg.org/wiki/HWAccelIntro this should work for amd, Nvidia, Intel, and any other opencl 1.2 compatible implementation. And come to think about it, the eselect-opencl dependency isn't needed (that dependency ensures that opencl 2.x is available, but ffmpeg only needs 1.2).